### ISSUE TITLE
keras TF 2.2 mileading error mesg

### ISSUE THREAD

**Issue by Vikas-kum** [awslabs/sagemaker-debugger#255]
Got this error mesg - 
 Disabling SMDebug as it does not support eager modefor TF versions 1.x

>>> import tensorflow as tf
>>> tf.__version__
‘2.2.0' 

I am using run_eagerly=True in model compile. 

model.compile(
      loss='sparse_categorical_crossentropy',
      optimizer=tf.keras.optimizers.Adam(0.001),
      metrics=['accuracy’], run_eagerly=True
  )
